brainscore_core/plugin_management/parse_plugin_changes.py

```diff
@@ -80,12 +110,13 @@ def parse_plugin_changes(changed_files: str, domain_root: str) -> dict:
 	"""
 	assert changed_files, "No files changed"
 	changed_files_list = changed_files.split()
-	changed_plugin_files, changed_non_plugin_files = separate_plugin_files(changed_files_list)	
+	changed_plugin_files, changed_non_plugin_files, changed_plugin_related_files = separate_plugin_files(changed_files_list)	
 
 	plugin_info_dict = {}
-	plugin_info_dict["modifies_plugins"] = False if len(changed_plugin_files) == 0 else True
+	plugin_info_dict["modifies_plugins"] = False if len(changed_plugin_files) + len(changed_plugin_related_files) == 0 else True
```

Comment:
so I tested that your code works fine, but to avoid future jumpscares
```suggestion
	plugin_info_dict["modifies_plugins"] = False if (len(changed_plugin_files) + len(changed_plugin_related_files)) == 0 else True
```